app/javascript/components/service-request-default/index.jsx

```diff
@@ -0,0 +1,138 @@
+import React from 'react';
+import MiqFormRenderer, { useFormApi } from '@@ddf';
+import PropTypes from 'prop-types';
+import { FormSpy } from '@data-driven-forms/react-form-renderer';
+import createSchema from './service-request-default.schema';
+
+// NOTE: parameters to be used as filters
+let daysAgo;
+let types;
+
+const requestTypeDefault = [
+  'provision_via_foreman',
+  'template',
+  'clone_to_vm',
+  'clone_to_template',
+  'orchestration_stack_retire',
+  'provision_physical_server',
+  'physical_server_firmware_update',
+  'service_retire',
+  'service_reconfigure',
+  'clone_to_service',
+  'vm_cloud_reconfigure',
+  'vm_migrate',
+  'vm_reconfigure',
+  'vm_retire',
+];
```

Comment:
I'm fine with this for now, but I'm wondering if this should be driven with an API OPTIONS call of some sort.  Thinking like OPTIONS /api/requests or maybe OPTIONS /api/service_requests (I'm not sure why we have 5 different /api/*_requests endpoints).